app/assets/stylesheets/style.scss

```diff
@@ -101,3 +101,7 @@ body {
 .inline {
   display: inline-block;
 }
+
+ table.center {
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Avoid qualifying class selectors with an element.